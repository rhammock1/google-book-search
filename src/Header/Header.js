import React from 'react';

const Header = function() {
  return (
    <Header>
      <h1>Google Book Search</h1>
    </Header>
  )
}

export default Header;