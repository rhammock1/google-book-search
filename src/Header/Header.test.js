import React, { createElement } from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import renderer from 'react-test-renderer';


describe('Header component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the UI as expected', () => {
    const tree = renderer.create(<Card title='randomTitle' content='filler content' />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})