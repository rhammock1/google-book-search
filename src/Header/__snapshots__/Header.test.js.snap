// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the UI as expected 1`] = `
<header>
  <h1>
    Google Book Search
  </h1>
</header>
`;
